lib/carrierwave-serializable/orm/activerecord.rb

```diff
@@ -53,8 +53,15 @@ def #{serialization_column}_changed?
         class_eval <<-RUBY, __FILE__, __LINE__+1
           def write_uploader(column, identifier)
             if self.class.serialized_uploader?(column)
-              serialized_field = self.send self.class.serialized_uploaders[column]
-              serialized_field[column.to_s] = identifier
+              serialized_field_name = self.class.serialized_uploaders[column].to_s
+
+              if serialized_field = self.send(serialized_field_name)
+                serialized_field[column.to_s] = identifier
+              else
+                # if the field happens to be nil, then all we have is a nil object. We need
+                # to 1.) initialize the hash and 2.) pass it to the field using the setter (=) method
+                self.send(serialized_field_name + '=', { column.to_s => identifier })
```

Comment:
Let's rather use this `self.send "#{serialized_field_name}=" column.to_s => identifier`